packages/example-inventory/002-connecting-ct/index.js

```diff
@@ -58,22 +62,31 @@ exports.helloWorld = function helloWorld(req, res) {
   } else {
     // Everything is okay.
     // Let's attempt to look up the inventory for that sku
-    const requestBuilder = createCTRequestBuilder()
-    const inventoryUri = requestBuilder.inventory.where(`sku="${req.body.sku}"`).build()
+    const requestBuilder = createCTRequestBuilder();
+    const inventoryUri = requestBuilder.inventory
+      .where(`sku="${req.body.sku}"`)
+      .build();
     const inventoryRequest = {
-      method: "GET",
-      uri: inventoryUri
-    }
-    const client =createCTClient()
-    client.execute(inventoryRequest).then(result => {
-      const results = result.body.results
-      if (!results.length) {
-        return res.status(404).json({error:'Inventory for SKU not found!'})
-      }
-      res.status(200).json(results)
-    }).catch(e => {
-      console.error(e)
-      res.status(400).json(e)
-    })
+      method: 'GET',
+      uri: inventoryUri,
+    };
+    const client = createCTClient();
+    client
+      .execute(inventoryRequest)
+      .then(result => {
+        const results = result.body.results;
+        if (!results.length) {
+          return res
+            .status(404)
+            .json({ error: 'Inventory for SKU not found!' });
+        }
+
+        return res.status(200).json(results);
+      })
+      .catch(e => {
+        // eslint-disable-next-line no-console
+        console.error(e);
+        res.status(400).json(e);
+      });
```

Comment:
All changes are due to formatting by prettier. No more hand formatting! Rules are enforced through ESLint. Just run `npm run format` or setup an editor integration for format on save. It's like gofmt.